package.json

```diff
@@ -23,21 +23,17 @@
     "url": "https://github.com/mojilla/material-snackbar-supplier/issues"
   },
   "homepage": "https://github.com/mojilla/material-snackbar-supplier#readme",
-  "peerDependencies": {
-    "@material-ui/core": "^3.0.1",
-    "@material-ui/icons": "^3.0.1",
-    "react": "^16.4.2"
+  "dependencies": {
+    "@material-ui/core": "^4.2.0",
+    "@material-ui/icons": "^4.2.1",
+    "react": "^16.8.6"
```

Comment:
Don't you think peerDependencies has benefited over dependencies in this case?
Anyone who attends to install it, he already installed material, react

Reply:
Ok, got it. I do not know about `peerDependencies` and its benefits. Thanks, I'll fix this.